src/main/java/org/opensearch/security/ssl/transport/SecuritySSLNettyTransport.java

```diff
@@ -103,6 +103,11 @@ public SecuritySSLNettyTransport(
         this.SSLConfig = SSLConfig;
     }
 
+    // This allows for testing log messages
+    public Logger getLogger() {
```

Comment:
maybe we can make the logger package-private since its static and is final , we don't need encapsulation.

Reply:
Sure will do.